tests/detect_machine.sh

```diff
@@ -30,8 +30,7 @@ case $(hostname -f) in
 
   s4-submit.ssec.wisc.edu) MACHINE_ID=s4 ;; ### s4
 
-  fe[1-4]) MACHINE_ID=jet_c ;; ### jet1-4
-  fe[5-8]) MACHINE_ID=jet ;; ### jet1-8
+  fe[1-8]) MACHINE_ID=jet ;; ### jet1-8
```

Comment:
@InnocentSouopgui-NOAA It seems to me the fe2 is still CentOS7 node. Do you know when it will be become Rocky8 node?

Reply:
That is correct, I noticed that as well, fe2 is still CentOS7. The team in charge of migrating to Rocky8 announced last week that they migrated everything (frontend nodes and partitions nodes) to Rocky8;
I don't know when they will migrate fe2 to Rocky8

Reply:
Okay, let's assume the jet final transition includes fe2.